pkg/management/component.go

```diff
@@ -624,6 +632,10 @@ func (c *component) UpdateUser(ctx context.Context, realmName, userID string, us
 	)
 
 	userRep = api.ConvertUpdatableToKCUser(user)
+	if user.Email.Value != nil {
+		// Keep previous email value. Update is managed with emailToValidate
+		userRep.Email = oldUserKc.Email
+	}
 
 	// Merge the attributes coming from the old user representation and the updated user representation in order not to lose anything
 	var mergedAttributes = make(kc.Attributes)
```

Comment:
How we ensure the phoneNumber/email is not directly updated ?

Reply:
Scenario seen together